plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/mapping/DefaultIdentifierMapping.java

```diff
@@ -32,19 +32,19 @@
     @Override
     public String fromRemoteSchemaName(String remoteSchemaName)
     {
-        return remoteSchemaName.toLowerCase(ENGLISH);
+        return remoteSchemaName == null ? null : remoteSchemaName.toLowerCase(ENGLISH);
```

Comment:
those are disturbing. Why would you need those. Common assumption is that arguments passed are not nullable. If we actually want to allow "empty" value we use `Optional`. There are rare cases when we want to allow nulls (mostly for performance reasons), but this needs to be very much justified and arg should be annotaded with `@Nullable`

Reply:
Ok, thanks - i wasn't sure whether we allow null parameter values, and JdbcOutputTableHandle's `catalogName` and `schemaName` constructor parameters are both annotated as `@Nullable`, which made me nervous that we just _happened_ to have avoided null values due to null checks elsewhere.

I can just remove these null checks.

Reply:
removed the null checks